pkgs/applications/networking/instant-messengers/signal-cli/default.nix

```diff
@@ -1,13 +1,13 @@
-{ stdenv, lib, fetchurl, makeWrapper, jre_headless, libmatthew_java, dbus, dbus_java }:
+{ stdenv, lib, fetchurl, makeWrapper, openjdk11_headless, libmatthew_java, dbus, dbus_java }:
```

Comment:
You need to replace all occurrences of jre_headless if this change is intentional.